jobs/config.json

```diff
@@ -170,10 +170,19 @@
       "sig-multicluster"
     ]
   },
+  "ci-ingress-gce-downgrade-e2e": {
+    "args": [
+      "true"
```

Comment:
fill this up?

Reply:
The logic to implement the upgrade and downgrade tests is still in progress so I don't want to block on that.